### ISSUE TITLE
[Defaults]: IMDB

### ISSUE THREAD

**Issue by dmfreck** [Kometa-Team/Kometa#1420]
### Link to File

plex_meta_manager.py

### Version Number

 1.19.0 

### What branch are you on?

master

### Describe the Bug

IMDb top 250 is out of order upon update. When PMM creates the collection from scratch it's ordered as it should be (i.e. Shawshank #1, etc). After subsequent runs, the collection is out of order.

**Comment by dmfreck** [comment]
Sorry, forgot to include my config.yml
[config.zip](https://github.com/meisnate12/Plex-Meta-Manager/files/11412766/config.zip)

**Comment by chazlarson** [comment]
This is a Plex issue that PMM cannot work around; items that are in more than one custom-ordered collection with a custom order do not appear in the order expected.  I suspect that if you look at your log you will see PMM telling Plex to put the items in the correct order, and Plex then drops the ball sometime later [not shown in the log].

If you could provide a log showing this problem we could confirm.

**Comment by dmfreck** [comment]
You are correct: I've watched the console output while PMM attempts to re-order the Top 250 and can see PMM telling Plex to move things around. However, PMM used to do this fine back before I switched to the "defaults" version of the IMDb Top 250 collection creation. I didn't know about the "items in more than one custom-ordered collection" bit, though, so maybe that's what is going on. Thanks for the response.
